File: src/kyu_5/not_very_secure/c/not_very_secure_test.c
#include <criterion/criterion.h>
#include <stdbool.h>

bool alphanumeric(const char* strin);

Test(the_multiply_function, should_pass_all_the_tests_provided) {
    cr_assert_eq(alphanumeric("hello_world"), false);
    cr_assert_eq(alphanumeric("PassW0rd"),    true);
    cr_assert_eq(alphanumeric("     "),       false);
    cr_assert_eq(alphanumeric("a"),           true);
}

File: src/kyu_5/not_very_secure/csharp/not_very_secure_test.cs
namespace Solution
{
  using NUnit.Framework;
  using System;
  using System.Collections.Generic;

  [TestFixture]
  public class Sample_Tests
  {
    private static IEnumerable<TestCaseData> testCases
    {
      get
      {
        yield return new TestCaseData("Mazinkaiser").Returns(true);
        yield return new TestCaseData("hello world_").Returns(false);
        yield return new TestCaseData("PassW0rd").Returns(true);
        yield return new TestCaseData("     ").Returns(false);
      }
    }

    [Test, TestCaseSource("testCases")]
    public bool Test(string str) => Kata.Alphanumeric(str);
  }
}